### PR TITLE
PR - Headermenu's integrations to Dev

### PR DESCRIPTION
**Pull request by carlsavila**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+  <app-menu><a routerLink="/menu"></a></app-menu>
+</header>
+
+<!-- MAIN -->
+<main class="app-main">
   <router-outlet></router-outlet>
 </main>
+
+<!-- FOOTER -->
+<footer class="app-footer">
```
Comment (Benja-san) [inline src/app/app.component.html]: this has no link with header

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
-    path: 'home',
-    component: HomeComponent,
-  },
-  {
-    path: '',
-    redirectTo: '/home',
-    pathMatch: 'full',
-  },
-
+  {    path: 'home', component: HomeComponent, },
```
Comment (Benja-san) [inline src/app/app.routes.ts]: french name for urls content, english for code

**Review by Benja-san — CHANGES_REQUESTED** [review]
I'll let the footer pass the Pr, but try to name your branches accordingly to your work done on this one

File: src/app/app.routes.ts (hunk 2 above)
Comment (carlsavila) [reply src/app/app.routes.ts]: done!

File: src/app/app.component.html (hunk 1 above)
Comment (carlsavila) [reply src/app/app.component.html]: Noted! Thanks.

**Review by Benja-san — CHANGES_REQUESTED** [review]
Change the poath into the menu template routerlink urls

**Comment by carlsavila** [comment]
CR Done! new path apply on the templates

**Comment by carlsavila** [comment]
"pull dev" and "push -u origin headermenu" Done!
Looks like there are no conflicts

**Review by Benja-san — APPROVED** [review]

**Pull request merged**